lib/gooddata/lcm/lcm2.rb

```diff
@@ -351,7 +365,7 @@ def perform(mode, params = {})
         end
 
         {
-          actions: actions.map(&:short_name),
+          actions: actions.map { |action| action.short_name },
```

Comment:
Style/SymbolProc: Pass `&:short_name` as an argument to `map` instead of a block.